DEPRECATIONS.md

```diff
@@ -0,0 +1,20 @@
+
+
+# Lagoon Project Deprecation Announcements
+
+## Overview
+This file is a constantly updated central tracker for deprecations across the suite of Lagoon products. As Lagoon continues to evolve, we occasionally need to replace, rename or retire existing processes, tools or configuration. Where this may impact a user's processes or procedures, we intend to outline a timeframe to allow any necessary changes to be made.
+
+Deprecations will be tracked by the release they are announced in, and then updated when the actual deprecation occurs. All deprecations should provide a rough timeline (in months or releases). Releases will normally only be listed here if they include planned deprecations.
+
+## Deprecation History
+All deprecations are listed below, with the most recent announcements at the top.
+
+### Lagoon v2.17.0
```

Comment:
I'm not sure how best to handle these linking to releases that may not exist yet?

Reply:
I want to avoid having to made a file change as part of a release - I think at the point we're collecting the PRs for a release, we know what the release is going to be called?

This table just announces the deprecations - we can link back to the deprecation announcement once we do the deprecation

Reply:
If you think its manageable otherwise, I'm happy to run with it